testplan/docs/openebs-enterprise/openebs/OpenEBS-Enterprise.md

```diff
@@ -0,0 +1,58 @@
+---
+id: OpenEBS-Enterprise 
+title: OpenEBS Enterprise
+sidebar_label: Test Strategy
+---
+------
+
+## OpenEBS EE (OpenEBS Enterprise)
+
+OpenEBS EE is an Enterprise version of OpenEBS certified by MayaData. This goes through rigorous testing in-addition to the testing covered as part of community edition <a href="http://openebs.ci/" target="_blank">openebs.ci</a> 
+
+This section covers the test strategy of the OpenEBS EE component in OEP.
+
+- Install and Upgrade - Install and upgrade of OpenEBS can be done in the following ways
+  - Helm
+  - Director - Testing of Install and Upgrade is covered in [Director](/docs/director/openebs-provisioning/plan) 
```

Comment:
do we have /docs/director/openebs-provisioning/plan  this link?

Reply:
yes this absolute link

Reply:
got the link. it is in e2emayadata.io